### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by Maxonxchik** [entropyxyz/community#121] [Get Test Funds] [answered]
### Address

5HRHDe3deSXHZjL4qhgXYqbPxryJ61xsubUrgxQDyrMMKYoC

**Comment by johnnymatthews** [comment, marked as answer]
Sent in block [53,750](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestnet.entropy.xyz#/explorer/query/0x85ddd2ca8a5cda82c5c81b71e5b280d1f6cbac53faf10a7127f1456259e07a0b)
